packages/opentelemetry-instrumentation/README.md

```diff
@@ -162,7 +162,7 @@ myPLugin.enable();
 Successor of loading plugins through TracerProvider "plugins" option.
 It also supersedes PluginLoader for node. The old configurations usually looks like
 
-### NODE - old way using TracerProvider
+### NODE - old way using TracerProvider - not available anymore
```

Comment:
Should we remove this is it is no longer available?

Reply:
I had hard times to decide whether to remove it or it should remain for some time so when people will be upgrading they can at least compare what has changed. That was the reason I left it but really not sure what is better from user perspective :/, can remove it as well

Reply:
I think it's ok to keep it for a while. But I think we should change it to something like `removed since 0.19.0`.